### DISCUSSION TITLE
Discord etc C2 functionality

### DISCUSSION THREAD

**Discussion by jacobvanderwyst** [BishopFox/sliver#1730] [Ideas]
For longevity purposes and my own interest in exploring the functionality of this tool. I think it would be an interesting idea to implement some form of discord bot C2 functionality in the way Dystopia-C2 does. I have been hearing about a lot of C2 frameworks moving towards this form of C2 and how it may be more effective long-term.

**Comment by pure-nomad** [comment]
heh heh [CordKit](https://github.com/pure-nomad/cordkit)

**Comment by moloch--** [comment]
I've long considered adding https://github.com/nikoksr/notify support, though I've yet to get around to it :) maybe this year I'll have time to work on Sliver again.
